app/models/agents/webhook_agent.rb

```diff
@@ -53,6 +54,10 @@ def receive_web_request(params, method, format)
       # check the verbs
       verbs = (interpolated['verbs'] || 'post').split(/,/).map { |x| x.strip.downcase }.select { |x| x.present? }
       return ["Please use #{verbs.join('/').upcase} requests only", 401] unless verbs.include?(method)
+      
+      # check the code
+      code = (interpolated['code'].present? && interpolated['code'].to_i) || 201
+      return ["Return code must not be empty", 401] unless !interpolated['code'].present? || code > 0
```

Comment:
I don't think this should error the request. Instead, it should be checked in `validate_options`.  If code isn't set, use the default of 201.

`code = (interpolated['code'].presence || 201).to_i`